src/screens/UserManagement/PermissionUtils.res

```diff
@@ -139,69 +139,85 @@ let getAccessValue = (~permissionValue: permissionType, ~permissionList) => {
+  apiKeyRead: NoAccess,
+  apiKeyWrite: NoAccess,
+  merchantAccountRead: NoAccess,
+  merchantAccountWrite: NoAccess,
+  merchantConnectorAccountRead: NoAccess,
+  merchantConnectorAccountWrite: NoAccess,
+  forexRead: NoAccess,
+  routingRead: NoAccess,
+  routingWrite: NoAccess,
+  disputeRead: NoAccess,
+  disputeWrite: NoAccess,
+  mandateRead: NoAccess,
+  mandateWrite: NoAccess,
+  customerRead: NoAccess,
+  customerWrite: NoAccess,
+  fileRead: NoAccess,
+  fileWrite: NoAccess,
+  analytics: NoAccess,
+  threeDsDecisionManagerWrite: NoAccess,
+  threeDsDecisionManagerRead: NoAccess,
+  surchargeDecisionManagerWrite: NoAccess,
+  surchargeDecisionManagerRead: NoAccess,
+  usersRead: NoAccess,
+  usersWrite: NoAccess,
 }
 
+// TODO: Refactor to not call function for every permission
 let getPermissionJson = permissionList => {
-  let getAccessValueFromPermission = permissionValue =>
-    getAccessValue(~permissionList, ~permissionValue)
-
   {
-    paymentRead: PaymentRead->getAccessValueFromPermission,
-    paymentWrite: PaymentWrite->getAccessValueFromPermission,
-    refundRead: RefundRead->getAccessValueFromPermission,
-    refundWrite: RefundWrite->getAccessValueFromPermission,
-    apiKeyRead: ApiKeyRead->getAccessValueFromPermission,
-    apiKeyWrite: ApiKeyWrite->getAccessValueFromPermission,
-    merchantAccountRead: MerchantAccountRead->getAccessValueFromPermission,
-    merchantAccountWrite: MerchantAccountWrite->getAccessValueFromPermission,
-    merchantConnectorAccountRead: MerchantConnectorAccountRead->getAccessValueFromPermission,
-    merchantConnectorAccountWrite: MerchantConnectorAccountWrite->getAccessValueFromPermission,
-    forexRead: ForexRead->getAccessValueFromPermission,
-    routingRead: RoutingRead->getAccessValueFromPermission,
-    routingWrite: RoutingWrite->getAccessValueFromPermission,
-    disputeRead: DisputeRead->getAccessValueFromPermission,
-    disputeWrite: DisputeWrite->getAccessValueFromPermission,
-    mandateRead: MandateRead->getAccessValueFromPermission,
-    mandateWrite: MandateWrite->getAccessValueFromPermission,
-    customerRead: CustomerRead->getAccessValueFromPermission,
-    customerWrite: CustomerWrite->getAccessValueFromPermission,
-    fileRead: FileRead->getAccessValueFromPermission,
-    fileWrite: FileWrite->getAccessValueFromPermission,
-    analytics: Analytics->getAccessValueFromPermission,
-    threeDsDecisionManagerWrite: ThreeDsDecisionManagerWrite->getAccessValueFromPermission,
-    threeDsDecisionManagerRead: ThreeDsDecisionManagerRead->getAccessValueFromPermission,
-    surchargeDecisionManagerWrite: SurchargeDecisionManagerWrite->getAccessValueFromPermission,
-    surchargeDecisionManagerRead: SurchargeDecisionManagerRead->getAccessValueFromPermission,
-    usersRead: UsersRead->getAccessValueFromPermission,
-    usersWrite: UsersWrite->getAccessValueFromPermission,
+    paymentRead: getAccessValue(~permissionValue=PaymentRead, ~permissionList),
```

Comment:
Why this change needed?

Reply:
There was a function inside a function and also we could have directly called the function .

Reply:
~permissionList writing unnecessarily

Reply:
Instead of calling the function for each permission individually, it could have been called once at the beginning of the function. This would reduce the need to iterate over the list of permissions multiple times.